packages/config/src/__tests__/ConfigParsing-test.ts

```diff
@@ -96,6 +96,7 @@ describe(getConfig, () => {
   // - generated `.expo` object is created and the language hint is added
   describe('language support', () => {
     beforeEach(() => {
+      delete process.env.EXPO_DEBUG;
```

Comment:
we should probably do this before every test in the expo-cli repo, maybe in global setup. i noticed that we set EXPO_DEBUG on ci but people may not have that set locally, which is i guess what you ran into here too

Reply:
i guess in general process.env should just have no contents unless we explicitly set them for specific tests